specificatie/openapi.yaml

```diff
@@ -28,24 +28,28 @@ paths:
           description: Zoek WOZ-objecten in eigendom van een bij het Handelsregister ingeschreven niet-natuurlijk persoon
           schema:
             type: string
+            pattern: ^[0-9]{9}$
             example: "857567433"
         - name: kvkNummer
           in: query
           description: Zoek WOZ-objecten in eigendom van een bij het Handelsregister ingeschreven maatschappelijke activiteit of een van de daaronder vallende ondernemingen en vestigingen
           schema:
             type: string
+            pattern: ^[0-9]{8}$
             example: "68727720"
         - name: adresseerbaarObjectIdentificatie
           in: query
           description: Zoek op de BAG identificatie van een adresseerbaar object (verblijfsobject, standplaats of ligplaats) waar het WOZ-object aan verbonden is of waar het object zijn aanduiding aan ontleent
           schema:
             type: string
-            example: "0226010000038820"
+            pattern: ^[0-9]{16}$
+            example: "0122260100000388"
```

Comment:
Is er een reden dat er een nieuwe/andere example is gespecificeerd?

Reply:
De oude example was hetzelfde als het example voor de nummeraanduiding. Dat leek mij geen realistisch voorbeeld. Meer zit er niet achter.

Reply:
een nummeraanduidingIdentificatie heeft op posities 5-6 "20". Bij nummeraanduidingIdentificatie zie ik als example '0226200000038820", wat dus een goed voorbeeld is voor een nummeraanduidingIdentificatie en ongelijk aan het example voor adresseerbaarObjectIdentificatie dat je nu gewijzigd hebt.
een adresseerbaar object heeft op posities 5-6 "01" (verblijfsobject), "02" (ligplaats) of "03" (standplaats)

dus "0226010000038820" is een goed voorbeeld voor een adresseerbaar object, maar een slecht voorbeeld voor een nummeraanduiding. "0122260100000388" zal voor gebruikers die een beetje bekend zijn met BAG identificaties direct herkenbaar zijn als niet-valide adresseerbaarObjectIdentificatie. Niet handig...